src/sayali/Assignment63.java

```diff
@@ -0,0 +1,34 @@
+/* WAP for 1. Count the prime number in range of 50 to 150
+2. Do sum of number in range of 50 to 150, only if number is odd & prime */
+
+package sayali;
+
+public class Assignment63 {
+
+	boolean checkPrime(int n) {
+		for (int i = 2; i < n / 2; i++) {
+			if (n % i == 0) {
+				return false;
+			}
+		}
+		return true;
+	}
+
+	void oddPrimeSum() {
+		int count = 0;
+		int sum = 0;
+		for (int num = 50; num <= 150; num++) {
```

Comment:
You have to pass 2 numbers as parameter, here 50 and 150 is just for reference, you have to make your code dynamic